spec.ttl

```diff
@@ -34,6 +34,15 @@ spec:Specification
   rdfs:label "Specification"@en ;
   rdfs:isDefinedBy spec: .
 
+spec:ClassOfProduct
+  a rdfs:Class ;
+  rdfs:label "Class of Product"@en .
```

Comment:
Considering `spec:ClassesOfProducts` exists and has been in use for sometime now, I don't think your suggestion is a great idea.